app/main/views/frameworks.py

```diff
@@ -51,7 +51,7 @@ def framework_dashboard(framework_slug):
                 [user['emailAddress'] for user in supplier_users['users'] if user['active']],
                 email_body,
                 current_app.config['DM_MANDRILL_API_KEY'],
-                'You have started your {} application'.format(framework['name']),
+                'You started a {} application'.format(framework['name']),
```

Comment:
Only problem with this language is it will require a "a"/"an" switch if we come up with a framework beginning with a vowel.